pkg/storage/tree/tree.go

```diff
@@ -108,6 +108,13 @@ func prepend(s []*treeNode, x *treeNode) []*treeNode {
 	return append(s, prev)
 }
 
+func prependBytes(s [][]byte, x []byte) [][]byte {
+	s = append(s, nil)
+	copy(s[1:], s)
+	s[0] = x
+	return s
+}
```

Comment:
I'm sure `prepend` can be done in the same way, without unnecessary checks, like:
```go
func prependTreeNode(s []*treeNode, x *treeNode) []*treeNode {
	s = append(s, nil)
	copy(s[1:], s)
	s[0] = x
	return s
}
```

Reply:
That's right. I added a commit for `prependTreeNode`.